### PR TITLE
Update message_engagement_events.md remove duplication of canvas_name

### PR DESCRIPTION
**Pull request by longchass**
### Summary

My PR clean the documentation, there is a column name canvas_name that gets duplicated in the braze message event content card impression

### Related PRs, issues, or features (optional)

- N/A

### Feature release date (optional)

- N/A

### Contributor checklist

- [X] I confirm that my PR meets the following:
    - I signed the [Contribution License Agreement (CLA)](https://www.braze.com/docs/cla).
    - My style and voice follows the [Braze Style Guide](https://docs.google.com/document/u/2/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub).
    - My content contains correct spelling and grammar.
    - All links are working correctly.
    - If I renamed or moved a file or directory, I set up [URL redirects](https://www.braze.com/docs/contributing/content_management/redirecting_urls) for each file.
    - If I updated or replaced an image, I did not remove the original image file from the repository. (For more information, see [Updating an image](https://www.braze.com/docs/contributing/content_management/images/#updating-an-image)).
    - If my PR is related to a paid SKU, third party, SMS, AI, or privacy, I have received written approval from Braze Legal.

### Submitting for review

If your PR meets the above requirements, select **Ready for review**, then add a reviewer:
- **Non-Braze contributors:** Add `braze-inc/docs-team` as the reviewer.
- **Braze employees:** Have any relevant subject matter experts (like engineers or product managers) review your work first, then add the [tech writer assigned to your specific vertical](https://confluence.braze.com/pages/viewpage.action?pageId=75039896). If you're not sure which tech writer to add, you can add `braze-inc/docs-team` instead. 

Thanks for contributing! We look forward to reading your work.

### CODE REVIEW

**Comment by longchass** [comment]
@cla-bot check

**Comment by longchass** [comment]
@braze-inc/docs-team
Seems like I can't add any reviewers

**Comment by josh-mccrowell-braze** [comment]
Hi @longchass - Sorry about that! We'll have to look into what the permissions are for folks to add reviewers--you _should_ be able to if you're a member of the Braze org. 

Are you trying to add reviewers from your team (if so, let us know who and we'll be happy to add them) or tag the Doc's team in for a review and merge?

**Comment by longchass** [comment]
Oh, my bad. I'm just a customer of Braze, went through the documentation and got some errors because of the duplicated column.

I believe the fix in the documentation is correct.

**Comment by josh-mccrowell-braze** [comment]
@longchass Ah! Thank you for clarifying! And for solving the mystery of why you can't add reviewers. Tagging the @braze-inc/docs-team is the right move in this case. 

Thank you for improving our documentation. We'll take it from here and get this verified with the product teams, then merged in to help other customers. 

Please let me know if you have any questions

**Review by froodian — APPROVED** [review]

File: _docs/_user_guide/data_and_analytics/braze_currents/event_glossary/message_engagement_events.md [hunk 1]
```diff
@@ -1414,7 +1414,6 @@ This event occurs when a user views a Content Card.
   "canvas_variation_name": (optional, string) name of the Canvas variation the user is in if from a Canvas,
   "canvas_step_id": (optional, string) ID of the step for this message if from a Canvas,
   "canvas_step_name": (optional, string) name of the Canvas step this event belongs to,
-  "canvas_name": (optional, string) name of the Canvas,
```
Comment (froodian) [inline _docs/_user_guide/data_and_analytics/braze_currents/event_glossary/message_engagement_events.md:1417]: yep i see this also appears just above the fold at 1412

**Review by rachel-feinberg — APPROVED** [review]
Lgtm!

**Pull request merged**